### PR TITLE
fix-NXP-29579-RetentionLabels-10.10 (Backport 10.10)

### PR DESCRIPTION
**Pull request by nmpcunha**
As an output of this work I would suggest the following:
- We should state somewhere (maybe [here](https://nuxeowiki.atlassian.net/wiki/spaces/NIG/pages/845643854/Backport+Procedures+and+Policies)?) that we need to create the entries in every file, specially on 10.10 because no crowdin job is running for that branch;
- We probably should try to have that in mind when reviewing PRs (specially backports);

### CODE REVIEW

**Comment by manonlumeau** [comment]
Some of the labels are still in Engish even if the translation is available on Master and so Crowdin. 

Some examples of inconsistencies: 
Spanish:
1010 example: https://github.com/nuxeo/nuxeo-web-ui/pull/1073/files#diff-c344a3b4e645043d68e618fad0aeb546800eca1b8b5bef1771242f5c3430b6c9R27
Same label on master: https://github.com/nuxeo/nuxeo-web-ui/blob/master/i18n/messages-es-ES.json#L28

File: i18n/messages-fr.json [hunk 1]
```diff
@@ -499,8 +514,13 @@
   "eventCategory.eventWorkflowCategory": "Workflow d'approbation",
   "eventCategory.NuxeoAuthentication": "Authentification",
   "eventCategory.relationNotificationCategory": "En attente de publication",
+  "eventType.afterMakeRecord": "Declared as a record",
```
Comment (manonlumeau) [inline i18n/messages-fr.json:517]: Translation on master: https://github.com/nuxeo/nuxeo-web-ui/blob/master/i18n/messages-fr.json#L560

File: i18n/messages-ar.json [hunk 2]
```diff
@@ -1,6 +1,11 @@
 {
   "activity.addedToCollection": "added to collection",
   "activity.auditLogRoute": "requested audit",
+  "activity.afterExtendRetention": "Retention period updated",
```
Comment (manonlumeau) [inline i18n/messages-ar.json:4]: See comment on main page about this file.

File: i18n/messages-es-ES.json [hunk 3]
```diff
@@ -55,13 +61,15 @@
...
   "app.document.fileDeleted": "Archivo eliminado",
+  "app.document.hold": "Document on legal hold",
   "app.document.locked": "Documento bloqueado",
   "app.document.removedFromClipboard": "Documento eliminado del portapapeles",
   "app.document.removedFromCollection": "Documento eliminado de la colección.",
   "app.document.removedFromFavorites": "Documento eliminado de favoritos.",
   "app.document.subscribed": "Ahora está siguiendo este documento.",
   "app.document.untrashed.error": "Error al restaurar el documento",
   "app.document.untrashed.success": "El documento se ha restaurado correctamente",
+  "app.document.unhold": "Legal hold removed",
```
Comment (manonlumeau) [inline i18n/messages-es-ES.json:72]: Translation on master: https://github.com/nuxeo/nuxeo-web-ui/blob/master/i18n/messages-es-ES.json#L105

**Review by manonlumeau — CHANGES_REQUESTED** [review]

**Comment by nmpcunha** [comment]
Thanks @manonlumeau for the review. Since this kind of work is a little bit error prone (copy/paste) it is always good to have other 👀  looking at it. Much appreciated 🙂 

Hope I've tackled all the issues found.

**Review by manonlumeau — APPROVED** [review]

**Review by mnixo — APPROVED** [review]

**Pull request merged**